packages/plugins/webpack/src/utils.ts

```diff
@@ -116,3 +116,16 @@ export const buildManifest = (
     { publicPath, assets: {} }
   );
 };
+
+export const removeQueryParams = (input: string) => {
```

Comment:
If it's duplicated 3 times it should probably be extracted somewhere, but I also don't know whether there's a suitable shared dependency all the usages could import from. The function is trivial enough that it's possibly not worth the additional effort (if there's no existing common RLL package that's available everywhere)

Reply:
I duplicated because there isn't a shared dep and I didn't think it was worth introducing one. In the past I've dealt with this by having internal dev deps that are compiled into the package. You need a build script capable of handing the externalisation logic though.